manifests/m/Microsoft/OpenJDK/11/11.0.15/Microsoft.OpenJDK.11.installer.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenJDK.11
+PackageVersion: 11.0.15
```

```suggestion
PackageVersion: 11.0.15.10
```
This must match Control Panel.

You'll also need to update the version folder from `11.0.15` to `11.0.15.10`